front/components/assistant/TagsFilterMenu.tsx

```diff
@@ -85,6 +85,11 @@ export const TagsFilterMenu = ({
             />
           }
         >
+          {filteredTags.length === 0 && (
```

Comment:
@tdraier FYI I added this text, otherwise it looked sad 😄
<img width="414" alt="Screenshot 2025-05-13 at 12 13 53" src="https://github.com/user-attachments/assets/4741a192-e7ea-4b5d-8774-04fb6a59c22c" />